### ISSUE TITLE
Missing line breaks in 'Spectre variant 2 and meltdown (own OS)'

### ISSUE THREAD

**Issue by pacoito123** [undergroundwires/privacy.sexy#31]
I don't think this script is supposed to run on a single line, as it exits out of the entire Command Prompt when run.

This is the default generated script **(as of v0.7.3)**:
```
reg add "HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\Memory Management" /v "FeatureSettingsOverrideMask" /t REG_DWORD /d 3 /f wmic cpu get name | findstr "Intel" >nul && ( reg add "HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\Memory Management" /v "FeatureSettingsOverride" /t REG_DWORD /d 0 /f ) wmic cpu get name | findstr "AMD" >nul && ( reg add "HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\Memory Management" /v "FeatureSettingsOverride" /t REG_DWORD /d 64 /f )
```
This is what works for me:
```
reg add "HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\Memory Management" /v "FeatureSettingsOverrideMask" /t REG_DWORD /d 3 /f
wmic cpu get name | findstr "Intel" >nul && ( reg add "HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\Memory Management" /v "FeatureSettingsOverride" /t REG_DWORD /d 0 /f )
wmic cpu get name | findstr "AMD" >nul && ( reg add "HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\Memory Management" /v "FeatureSettingsOverride" /t REG_DWORD /d 64 /f )
```

Simple fix, but hope this helps :^).

**Comment by undergroundwires** [comment]
Hi 👋, it actually got broken in 0.7.3, hats off for detecting it so quick and thanks for the feedback with a constructive solution. It should be fixed now in 0.7.4 that's up.

**Comment by undergroundwires** [comment]
Hi 👋, it actually got broken in 0.7.3 (the day before), hats off for detecting it so quick and thanks for the feedback with a constructive solution. It should be fixed now in [0.7.4](https://github.com/undergroundwires/privacy.sexy/releases/tag/0.7.4) that's up.
